include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -969,6 +969,56 @@ namespace internal
 
   class MGTwoLevelTransferImplementation
   {
+    template <int dim, typename Number>
+    static std::shared_ptr<const Utilities::MPI::Partitioner>
+    create_coarse_partitioner(
+      const DoFHandler<dim> &                  dof_handler_coarse,
+      const dealii::AffineConstraints<Number> &constraint_coarse,
+      const unsigned int                       mg_level_coarse)
+    {
+      const IndexSet locally_active_dofs =
+        (mg_level_coarse == numbers::invalid_unsigned_int) ?
+          DoFTools::extract_locally_active_dofs(dof_handler_coarse) :
+          DoFTools::extract_locally_active_level_dofs(dof_handler_coarse,
+                                                      mg_level_coarse);
+
+      std::vector<types::global_dof_index> locally_relevant_dofs_temp;
+
+      for (const auto i : locally_active_dofs)
+        {
+          locally_relevant_dofs_temp.emplace_back(i);
+
+          if (constraint_coarse.is_constrained(i))
+            {
+              const auto constraints =
+                constraint_coarse.get_constraint_entries(i);
+
+              if (constraints)
```

Comment:
Isn't this check doing the same as the one in line 991 above?

Reply:
I always had the impression that you need to call `is_constrained()` before `get_constraint_entries()`. But after taking a detailed look at:

https://github.com/dealii/dealii/blob/3cd782f30906812dfcff541aff3d87b66a33806a/include/deal.II/matrix_free/dof_info.templates.h#L143-L147

 I came to the realization that it is indeed not need :+1: